content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -39,11 +39,9 @@ title: Dolt System Tables
 
 `dolt_branches` contains information about branches known to the database.
 
-Branches can be created, modified, or deleted with `INSERT`, `UPDATE`,
-or `DELETE` statements.
-
 Because the branch information is global to all clients, not just your
-session, it's recommended to lock the table before modifying it.
+session, `dolt_branches` system table is read-only. Branches can be created
+or deleted with [`DOLT_BRANCH()` stored procedure](dolt-sql-procedures.md#dolt_branch).
```

Comment:
(minor) Just adding the definite article in front of dolt_branch to make it read a little smoother. 

```suggestion
or deleted with the [`DOLT_BRANCH()` stored procedure](dolt-sql-procedures.md#dolt_branch).
```